terra/tests.py

```diff
@@ -365,6 +366,7 @@ def test_unit_list_loads(self):
         self.assertTemplateUsed(response, "terra/unit_list.html")
 
 
+@tag("trouble")
```

Comment:
@aprigge I don't think this is the best solution.  Excluding this test via the tag for Travis may work, but it still fails when run locally (unless developers also exclude when running there), so it's potentially inconsistent.

Arguably we could get rid of the whole test case as this is for CSV loading, which probably will never be needed again.  But that's not ideal either.

The accepted answer on the SO article you cited would be better.  Looking at https://docs.djangoproject.com/en/2.2/topics/testing/tools/ it looks like it may be as simple as adding a class method `tearDownClass` to the test case and calling that on `super`... though a quick test locally did not work, same error.

So, see if you can improve on this?  But if not, I can merge and deploy at the end of today so this moves forward, but there should be a new Jira ticket to try to fix the technical debt added by skipping this test case.